docs/services/OneDrive.md

```diff
@@ -1,84 +1,60 @@
 ---
 title: OneDrive Service
-author: nmetulev
+author: tgoodhew
 description: The OneDrive Service provides a simple way to access resources on either OneDrive or OneDrive for Business (Office 365).
 keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, OneDrive
 ---
 
 # OneDrive Service
 
-The **OneDrive** Service provides a simple way to access resources on either OneDrive or OneDrive for Business (Office 365).  You can:
+The **OneDrive** Service provides an easy to use service helper for the [OneDrive Developer Platform](https://docs.microsoft.com/en-us/onedrive/developer/) that uses the [Microsoft Graph](https://developer.microsoft.com/en-us/graph/docs/concepts/overview). The new OneDrive API is REST API that brings together both personal and work accounts in a single authentication model. The OneDrive Service helps you:
 
-* Authenticate using Microsoft Account, Active Directory or silently using OnlineId
-* Access OneDrive folders and files in the same consistent way as local StorageFolder and StorageFile
+* Initialize and authenticate with a common set of objects
+* Access OneDrive, OneDrive for Business, SharePoint document libraries, and Office Groups, to allow your app the flexibility to read and store content in any of these locations with the same code
```

Comment:
Would it be useful to have links to these other API docs?

Reply:
It's all just the OneDrive API or are you asking for links to the local storage docs?